### ISSUE TITLE
Add a test case to an existing test suite

### ISSUE THREAD

**Issue by dmcassel** [marklogic-community/ml-gradle#644]
The `mlGenerateUnitTestSuite` task will create a new test suite, but if you give it the name of an existing suite it will overwrite the setup and teardown modules. Allow generation of a new test case. This could be done a couple ways:

1. Modify `mlGenerateUnitTestSuite` so that if the test suite already exists, it creates the new test case and skips all the setup/teardown. 
2. Create a new task for generating only test cases.

**Comment by rjrudin** [comment]
@dmcassel I'm wondering about the value of this now - in practice, I just copy/paste an existing test, as that is likely to already be importing other libraries that are helpful for my tests. That seems easier than having to remember a Gradle task name and type out the correct arguments.

**Comment by dmcassel** [comment]
@rjrudin fair enough. Is it worth having the `mlGenerateUnitTestSuite` task see that the target directory already exists and decline to go further, to avoid overwriting things? Or good enough to rely on git's ability to restore?
